src/App.js

```diff
@@ -1,40 +1,39 @@
-import React, { Component } from "react";
-import Button from '@mui/material/Button';
-import SignUp from "./SignUpContainer";
+import "./App.scss";
+import React from "react";
+import { Routes, Route, BrowserRouter } from "react-router-dom";
+import Listings from "./containers/landlord/listings";
+import ListingInfo from "./containers/landlord/listingInfo";
+import ListingSearch from "./containers/landlord/listingSearch";
+import ApplicationInfo from "./containers/landlord/applicationInfo";
 
-class App extends Component {
-  constructor() {
-    super();
-    this.state = {
-      name: "React",
-      showSignUp: false,
-    };
-    this.hideComponent = this.hideComponent.bind(this);
-  }
-
-  hideComponent(name) {
-    console.log(name);
-    switch (name) {
-      case "showSignUp":
-        this.setState({ showSignUp: !this.state.showSignUp });
-        break;
-      default:
-    }
-  }
-
-  render() {
-    const { showSignUp } = this.state;
-    return (
-      <div>
-        <div>
-          <Button onClick={() => this.hideComponent("showSignUp")}>
-            Sign Up
-          </Button>
-        </div>
-        {showSignUp && <SignUp/>}
+function App() {
+  return (
+    <BrowserRouter>
+      <div className="App">
+        {/* <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header> */}
+        {/* <div>Here</div> */}
```

Comment:
Can we remove the comment?

Reply:
We will need to replace this comment with Navbar, hence kept the comment. Will remove in the next PR